governance/votes/Executive vote - June 13, 2024.md

```diff
@@ -0,0 +1,169 @@
+		
+| Delegate    | Amount (MKR) | Address                                                                                                               |
+|-------------|--------------|-----------------------------------------------------------------------------------------------------------------------|
+| BLUE        | 41.67        | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
+| Cloaky      | 41.67        | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) |
+| JuliaChang  | 41.67        | [0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7](https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7) |
+| Pipkin      | 34.26        | [0x0E661eFE390aE39f90a58b04CF891044e56DEDB7](https://etherscan.io/address/0x0E661eFE390aE39f90a58b04CF891044e56DEDB7) |
+| Byteron     | 12.50        | [0xc2982e72D060cab2387Dba96b846acb8c96EfF66](https://etherscan.io/address/0xc2982e72D060cab2387Dba96b846acb8c96EfF66) |
+| BONAPUBLICA | 11.11        | [0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3) |
+| WBC         | 11.11        | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) |
+| PBG         | 8.49         | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) |
+| Rocky       | 3.70         | [0xC31637BDA32a0811E39456A59022D2C386cb2C85](https://etherscan.io/address/0xc31637bda32a0811e39456a59022d2c386cb2c85) |
+| vigilant    | 2.78         | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) |
+| UPMaker     | 1.85         | [0xbB819DF169670DC71A16F58F55956FE642cc6BcD](https://etherscan.io/address/0xbB819DF169670DC71A16F58F55956FE642cc6BcD) |
+		
+### Q2 2024 AVC Member Compensation
+		
+- **Authorization:** [Atlas 2.5.10 prior to its removal by MIP102c2-SP34](https://github.com/makerdao/mips/pull/1100/files#diff-bad206bdb2371d68c4940c519b4cf8817b59781422c6d7c0d8da48ab24decb33L535-L565)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/avc-member-participation-rewards-q2-2024/24442)
+		
+If this executive proposal passes, the following distributions totaling **100.08 MKR** will be made for Q2 2024 AVC member compensation. This has been pro-rated to two months following the most recent amendments to The Atlas that removed the AVC Member role.
+		
+| AVC Member Name | Address                                                                                                               | AVC                | Amount (MKR) | 
+|-----------------|-----------------------------------------------------------------------------------------------------------------------|--------------------|--------------|
```

Comment:
We have an extra column here

Reply:
we don't see an extra column

Reply:
The table should have three columns, but this line defines four:

> |1-----------------|2-----------------------------------------------------------------------------------------------------------------------|3--------------------|4--------------|

I'm pointing this out because I don't know how the engine will render it